### DISCUSSION TITLE
Default Theme

### DISCUSSION THREAD

**Discussion by Alice-Deane** [pglombardo/PasswordPusher#1059] [Q&A] [answered]
Hello everyone,

Is there a way to disable the dark theme to only authorize the default "bright" one ?

Also I am unable to change theme via the env variable (Docker Compose) : 
![image](https://user-images.githubusercontent.com/104282193/234589637-fe88c1b4-b1c9-48ec-9659-34aee1b2f0a6.png)

Many thanks !

**Comment by pglombardo** [comment, marked as answer]
Hi @Alice-Deane - set this environment variable too `PWP_PRECOMPILE=true`.  This rebuilds the CSS on container boot.  Documentation is [here](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#configuring-a-theme).

There is no way to disable the dark version with the default theme.

I hope this helps!

**Comment by Alice-Deane** [comment]
It works perfectly ! 
Thanks :)
